hyp3_metadata/__main__.py

```diff
@@ -4,7 +4,8 @@
 from dateutil.parser import parse as dt_parse
 
 from hyp3_metadata import __version__, create_metadata_file_set_rtc
-from hyp3_metadata.util import SUPPORTED_DEMS, populate_example_data
+from hyp3_metadata.rtc import populate_example_data
```

Comment:
hmm, handleing different products here might be annoying... Could be a good use-case for subparsers:

```
python -m hyp3_metadata rtc [RTC_OPTIONS]
python -m hyp3_metadata insar [INSAR_OPTIONS]
```
Or via the entrypoint game we setup in hyp3-gamma

Reply:
Yeah, I haven't gotten to dealing with main yet, That seems like a good solution, I just haven't looked at it yet.